packages/wonder-blocks-link/src/components/__tests__/link.test.js

```diff
@@ -325,38 +298,38 @@ describe("Link", () => {
     describe("raw events", () => {
         test("onKeyDown", () => {
             // Arrange
-            const keyMock = jest.fn();
-            const wrapper = mount(
-                <Link href="/" onKeyDown={keyMock}>
+            let keyCode;
+            render(
+                <Link href="/" onKeyDown={(e) => (keyCode = e.keyCode)}>
                     Click me!
                 </Link>,
             );
 
             // Act
-            wrapper.find(Link).simulate("keydown", {keyCode: 32});
+            const link = screen.getByText("Click me!");
+            // eslint-disable-next-line testing-library/prefer-user-event
+            fireEvent.keyDown(link, {keyCode: 32});
 
             // Assert
-            expect(keyMock).toHaveBeenCalledWith(
-                expect.objectContaining({keyCode: 32}),
-            );
+            expect(keyCode).toEqual(32);
         });
 
         test("onKeyUp", () => {
             // Arrange
-            const keyMock = jest.fn();
-            const wrapper = mount(
-                <Link href="/" onKeyDown={keyMock}>
+            let keyCode;
+            render(
+                <Link href="/" onKeyUp={(e) => (keyCode = e.keyCode)}>
                     Click me!
                 </Link>,
             );
 
             // Act
-            wrapper.find(Link).simulate("keydown", {keyCode: 32});
+            const link = screen.getByText("Click me!");
+            // eslint-disable-next-line testing-library/prefer-user-event
+            fireEvent.keyUp(link, {keyCode: 32});
```

Comment:
**question:** why did this change to a keyup from a keydown?

Reply:
I think the previous test was giving false positives as we are intending to test `keyUp` and the previous version was testing `keyDown` instead.